packages/typescript-estree/src/convert-comments.ts

```diff
@@ -21,8 +21,8 @@ export function convertComments(
     (_, comment) => {
       const type =
         comment.kind == ts.SyntaxKind.SingleLineCommentTrivia
-          ? 'Line'
-          : 'Block';
+          ? AST_TOKEN_TYPES.Line
+          : AST_TOKEN_TYPES.Block;
```

Comment:
FYI I raised #1503

Reply:
before there was actually 2 types for comments, Comment and Token with AST_TOKEN_TYPES

this change unites both of them (before this was valid)